src/GovUk.Education.ExploreEducationStatistics.Data.Services.Tests/Utils/TableBuilderUtilsTests.cs

```diff
@@ -0,0 +1,162 @@
+﻿#nullable enable
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.CollectionUtils;
+using static GovUk.Education.ExploreEducationStatistics.Data.Services.Utils.TableBuilderUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Services.Tests.Utils
+{
+    public class TableBuilderUtilsTests
+    {
+        [Fact]
+        public void MaximumTableCellCount_NoFilters_SingleIndicatorLocationAndTimePeriod()
+        {
+            /*
+             * 2011  Bury  cell1
+             */
+            Assert.Equal(1, MaximumTableCellCount(1, 1, 1));
+        }
+
+        [Fact]
+        public void MaximumTableCellCount_NoFilters_MultipleIndicators()
+        {
+            /*
+             * 2011  Bury  cell1  cell2  cell 3
+             */
+            Assert.Equal(3, MaximumTableCellCount(3, 1, 1));
+        }
+
+        [Fact]
+        public void MaximumTableCellCount_NoFilters_MultipleLocations()
+        {
+            /*
+             * 2011  Bury  cell1
+             * 2011  Kent  cell2
+             * 2011  York  cell3
+             */
+            Assert.Equal(3, MaximumTableCellCount(1, 3, 1));
```

Comment:
Might be useful for the readability of these various tests to use named parameters, like:

```
        [Fact]
        public void MaximumTableCellCount_NoFilters_MultipleLocations()
        {
            /*
             * 2011  Bury  cell1
             * 2011  Kent  cell2
             * 2011  York  cell3
             */
            Assert.Equal(
                 3, 
                 MaximumTableCellCount(
                     indicatorCount: 1, 
                     locationCount: 3, 
                     timePeriodCount: 1
                 )
            );
        }
```

Reply:
Dodgy formatting optional :)

Reply:
Good suggestion, I've made this change now, thanks.